lib/dom.js

```diff
@@ -1139,10 +1139,11 @@ function needNamespaceDefine(node, isHTML, visibleNamespaces) {
  * Well-formed constraint: No < in Attribute Values
  * The replacement text of any entity referred to directly or indirectly in an attribute value must not contain a <.
  * @see https://www.w3.org/TR/xml/#CleanAttrVals
- * @see https://www.w3.org/TR/xml/#NT-AttValue
+  * @see https://www.w3.org/TR/xml/#NT-AttValue
+  * @see https://www.w3.org/TR/xml/#AVNormalize
```

Comment:
nit: consistent indenting

```suggestion
 * @see https://www.w3.org/TR/xml/#NT-AttValue
 * @see https://www.w3.org/TR/xml/#AVNormalize
```